apps/pipelines/nodes/nodes.py

```diff
@@ -454,10 +453,14 @@ class RouterNode(RouterMixin, Passthrough, HistoryMixin):
     )
 
     def _process_conditional(self, state: PipelineState, node_id=None):
+        default_keyword = self.keywords[self.defaultKeywordIndex] if self.keywords else None
         prompt = OcsPromptTemplate.from_messages(
-            [("system", self.prompt), MessagesPlaceholder("history", optional=True), ("human", "{input}")]
+            [
+                ("system", f"{self.prompt}\nThe default routing destination is: {default_keyword}"),
```

Comment:
Why was this necessary?

Reply:
without this, the default keyword is not used-- and then what's passed in https://github.com/dimagi/open-chat-studio/pull/1486/files#diff-9ff598a128c35d264a8848a5b193d20e69a960eb1e96f69b7003ed197d2b158bR479 is always the first keyword rather than the default

Reply:
right, so at first it seems like it would return None, but without modifying the prompt it always returns the first keyword